about.html

```diff
@@ -0,0 +1,146 @@
+    <title>The Collision Summit 2023</title>
+    <link rel="stylesheet" href="main.css" />
+  </head>
+  <body>
+    <header id="social-header">
+      <nav class="hamb-menu">
+        <ul>
+          <li><a href=""><img src="images/social/twitter-16.png" alt="twitter"></a></li>
+          <li><a href=""><img src="images/social/facebook-16.png" alt="facebook"></a></li>
+          <li><a href="#logout">Logout</a></li>
+          <li><a href="#mypage"> My Page</a></li>
+          <li><a href="#Portuguese"> Portuguese</a></li>
+        </ul>
+      </nav>
+    </header>
+    <header>
+      <div class="logo">
+        <a href="index.html"
+          ><img src="images/Logo/logo.avif" alt="logo"
+        /></a>
+      </div>
+      <nav class="hamb-menu">
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#Contact"> News</a></li>
+        </ul>
+      </nav>
+      <button type="button" id="header-button">Collision Campaign</button>
+    </header>
+    <div class="popup-menu">
+      <div class="menu-button">
+        <span class="hamburger"></span>
+        <span class="hamburger"></span>
+        <span class="hamburger"></span>
+      </div>
+      <ul>
+        <li>
+          <a href="index.html">Home</a>
+        </li>
+        <li><a href="about.html">About</a>
+        </li>
+        <li><a href="#Contact">News</a>
+        </li>
+      </ul>
+    </div>
+    <main class="main-content">
+      <section id="about-main">
+        <div>
+          <h1>"Hello! Tech World"</h1>
+          <h2>WEB Summit 2023</h2>
+          <hr class="solid gone" />
+          <p id="introduction">
+            Web Summit events have gathered half a million
+            people across Web Summit in Lisbon, Collision in
+            North America, and RISE in Asia since its
+            beginnings as a 150-person conference in Dublin
+            in 2009.
+            <span id="more-introduction"
+              >Web Summit’s mission has been to create
+              software that enables meaningful connections
+              between the CEOs, founders, investors, media,
+              politicians and cultural figureheads who are
+              reshaping the world.</span
+            >
+          </p>
+          <h3>
+            Please contact us per email for any further questions about the WEB SUMMIT 2023!
+          </h3>
+          <p id="email">info@torontosummit.com</p>
+        </div>
+      </section>
+      <section id="logo-reveal">
+        <h2>WEB SUMMIT 2023</h2>
+        <hr class="solid" />
+        <p>
+          The Global Summit brings together the community of experts and academics,
+          who compromise Collisions affiliate network in a different country every 
+          two years. This year, we hope to extend our invitation list including 
+          organisation, individuals who want to work with us on shared projects that
```

Comment:
Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design and the fonts).

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/164414192-2eb8d5ce-08b1-4f65-986b-1eab2d8828d6.png) | ![image](https://user-images.githubusercontent.com/10332499/164414273-db459b02-1c73-4eeb-a922-f33da940dbc4.png)

Reply:
resolved